### ISSUE TITLE
add to base16 collection

### ISSUE THREAD

**Issue by tgy** [bluz71/vim-moonfly-colors#9]
would it possible to add this theme to the base16 collection that a good amount of people is already using?

this would make it possible to seamlessly use this theme for many users who already rely on base16 and the tools built around it

**Comment by bluz71** [comment]
Hello,

Looking at the [base16](https://github.com/chriskempson/base16) project and their philosophy I don't believe moonfly is an appropriate theme to add to that collection.

Whilst it is true that moonfly currently fits within the 16 color guideline, that is a limit that could well change as time goes by.

moonfly defined 16 colors and used a bunch more from the `xterm-256color` palette primarily because I was a long-term Xterm user (not a true-color terminal) which meant I created a Vim `cterm` theme in parallel to the `gui` theme. Over the past couple years I ditched Xterm for modern true-color terminals and more recently I have switched over to using `termguicolors` which means I no use the moonfly `cterm` theme at all (but it still remains as part of this Vim theme for compatibility reasons).

My other [nightfly](https://github.com/bluz71/vim-nightfly-guicolors) theme is a GUI-only theme which uses more than 16 colors.

Eventually I could envisage doing the same here with moonfly, maybe in 2 or 3 years, that being to ditch `cterm` theme entirely and expanding the color palette beyond 16 colors + a few `xterm-256colors`. Maybe, maybe not. I want the freedom to move beyond 16 colors if that is how this theme evolves.

A bit long-winded, but philosophically moonfly really is not a base16 theme, hence it should not be part of that collection.

Best regards.
